metrics/seqeval/seqeval.py

```diff
@@ -106,8 +117,32 @@ def _info(self):
             reference_urls=["https://github.com/chakki-works/seqeval"],
         )
 
-    def _compute(self, predictions, references, suffix=False):
-        report = classification_report(y_true=references, y_pred=predictions, suffix=suffix, output_dict=True)
+    def _compute(
+        self,
+        predictions,
+        references,
+        suffix: bool = False,
+        scheme: Optional[str] = None,
+        mode: Optional[str] = None,
+        sample_weight: Optional[List[int]] = None,
+        zero_division: Union[str, int] = "warn",
+    ):
+        if scheme is not None:
+            try:
+                scheme_module = importlib.import_module("seqeval.scheme")
+                scheme = getattr(scheme_module, scheme)
```

Comment:
Why not import seqeval at the top of the file and then do
```python
scheme = getattr(seqeval.schema, scheme)
```
?

instead of catching AttributeError you can do
```python
scheme = getattr(seqeval.schema, scheme, None)
if scheme is None:
    raise ...
```

Reply:
`None` is a valid value for scheme parameter

So in that case it will look like this
```python
if sheme is not None:
    scheme = getattr(seqeval.scheme, scheme, None)
    if scheme is None:
        raise ...
```
Which I find confusing, personally, because of two `if` clauses, while catching an exception explicitly seems to be clearer to me. However, I don't feel strongly about that, so if you think that `getattr(seqeval.scheme, scheme, None)` is better, I'll be happy to make the change.

Reply:
Ok I see.
Let's keep it this way then or we may end up with something confusing. Thanks for the explanation !